.vscode/tasks.json

```diff
@@ -42,6 +44,25 @@
       },
       "problemMatcher": []
     },
+    {
+      "label": "Run tests for integration",
+      "detail": "Run tests for a given integration.",
+      "type": "shell",
+      "command": "${command:python.interpreterPath} -m pytest --timeout=10 ./tests/components/${input:integrationName}/",
```

Comment:
A very similar task already exists, did you check it before this PR?
https://github.com/home-assistant/core/blob/56545dacb00cf26457723653ae85de64eb3c8986/.vscode/tasks.json#L89-L92

Reply:
I don't really understand what is the difference with the new task, and I think this PR should be closed

Reply:
The difference is that the new task only executes the tests, whereas the existing task executes tests with coverage.

If executing test-only is not useful, I wonder why there are two tasks for it already.

Reply:
I don't think we need an additional task to run the integration tests without coverage. I'd rather add coverage parameters to the other pytest tasks and update the name of the coverage task to `Run tests for integration`.